### PR TITLE
[RFC] MapboxNavigationViewportDataSource logs

### PR DESCRIPTION
**Pull request by DzmitryFomchyn**
### Description
Logs for MapboxNavigationViewportDataSource

### CODE REVIEW

File: libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt [hunk 1]
```diff
@@ -382,6 +392,8 @@ class MapboxNavigationViewportDataSource(
             cameraForFollowing = followingCameraOptions,
             cameraForOverview = overviewCameraOptions
         )
+
+        Log.d("MapZoomingTest", "MapboxNavigationViewportDataSource.evaluate(). options: $options, resulting viewportData: $viewportData")
```
Comment (dzinad) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:396]: Why do you need to log options? You'll have them in viewportData.

Comment (dzinad) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:396]: Let's also log cameraState in the beginning of the method, as it is used in our calculations.

Comment (dzinad) [inline libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt:396]: Let's also log:
1. `pointsToFrameOnCurrentStep`, `targetLocation`, `additionalPointsToFrameForFollowing` and `pointsToFrameAfterCurrentStep` in `updateFollowingData`;
2. `simplifiedRemainingPointsOnRoute`, `targetLocation`, `additionalPointsToFrameForOverview` in `updateOverviewData`.

My camera intuition says the problem is somewhere in here, so it'd be nice to have as many logs as possible.

Comment (DzmitryFomchyn) [reply libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt]: That'd be easier to use the same options later when we continue investigation. But agree, that they're not really needed here.

**Pull request closed without merging**